src/tools/schema2kotlin.ts

```diff
@@ -330,12 +334,9 @@ abstract class Abstract${particle.name} : ${this.opts.wasm ? 'WasmParticleImpl'
     for (const connection of particle.connections) {
       connection.direction = 'reads writes';
       const handleName = connection.name;
-      // TODO(b/157598151): Update HandleSpec from hardcoded single EntitySpec to
-      //                    allowing multiple EntitySpecs for handles of tuples.
-      const entityType = SchemaNode.singleSchemaFullName(connection, nodes);
-      handleSpecs.push(this.handleSpec(handleName, entityType, connection));
-      const interfaceType = this.handleInterfaceType(connection, nodes, true);
+      const interfaceType = this.handleInterfaceType(connection, nodes, false /* not particle scope */);
```

Comment:
This is usually: `/* particleScope= */ false`. Unlike Java, I don't believe that format has any special meaning in TS, but using the exact variable name seems like a good idea at the very least.

Reply:
Done :)